lib/vty.c

```diff
@@ -3608,9 +3613,10 @@ int vty_mgmt_send_get_config(struct vty *vty, Mgmtd__DatastoreId datastore,
 				      num_req);
 
 	if (ret != MGMTD_SUCCESS) {
-		zlog_err("Failed to send GET-CONFIG to MGMTD for req-id %llu.",
-			 (unsigned long long)vty->mgmt_req_id);
-		vty_out(vty, "Failed to send GET-CONFIG to MGMTD!");
+		zlog_err(
+			"Failed to send GET-CONFIG to MGMTD for req-id %llu.",
+			(unsigned long long)vty->mgmt_req_id);
```

Comment:
Why not `uint64_t` everywhere for `unsigned long long`?

Reply:
Simply coz %llu takes unsigned long long :) Now I know uint64_t is also unsigned long long. But I think we saw a build failures on some platforms and had to go for 'unsigned long long' instead.

Reply:
Done.